### PR TITLE
support name props for Radio and custom tagName for Radio.Group

### PR DESCRIPTION
**Pull request by bojoyzhou**

### CODE REVIEW

File: src/radio/radio-group.jsx [hunk 1]
```diff
@@ -39,6 +39,10 @@ class RadioGroup extends Component {
          * radio group的默认值
          */
         defaultValue: PropTypes.oneOfType([PropTypes.string, PropTypes.number, PropTypes.bool]),
+        /**
+         * 设置标签类型
+         */
+        component: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
```
Comment (guanpu) [inline src/radio/radio-group.jsx]: Would you please add unit test to cover the `PropTypes.func` case?

**Comment by guanpu** [comment]
I don't see any problem, except that we need tests to cover the case when 'component' is `PropType.func` . And it would be better to describe a little bit on what problem this feat address, or in other words, why/when we need to customized a container type of a radio component. 

Anyway, I'm glad to see such a good contribution.

**Comment by bojoyzhou** [comment]
image
In my case, `ul` or `tbody` is better as a container type rather than the default `div`

File: src/radio/radio-group.jsx (hunk 1 above)
Comment (bojoyzhou) [reply src/radio/radio-group.jsx]: added in 3a80fe7

**Pull request merged**